src/Components/Utils/Utils.tsx

```diff
@@ -17,7 +17,11 @@ export const isQuestionHideCorrect = (
     question: Question,
     hideCorrect: boolean
 ) => {
-    return question.correct === hideCorrect;
+    if (hideCorrect) {
+        return question.correct !== hideCorrect;
+    } else {
+        return true;
+    }
```

Comment:
not to the scope of the nav bar, why is this here? this should have been its own pr or in the pr for refactors.